system/services/rulesEngine/RulesEngineAutoPresideObjectExpressionGenerator.cfc

```diff
@@ -49,17 +49,14 @@ component {
 		if ( IsBoolean( propertyDefinition.autofilter ?: "" ) && !propertyDefinition.autofilter ) {
 			return [];
 		}
-		if ( ( propertyDefinition.formula ?: "" ).len() ) {
-			return [];
-		}
 
 		var isRequired   = IsBoolean( propertyDefinition.required ?: "" ) && propertyDefinition.required;
 		var propType     = propertyDefinition.type ?: "string";
 		var relationship = propertyDefinition.relationship ?: "";
 		var relatedTo    = propertyDefinition.relatedTo ?: "";
 		var expressions  = [];
 
-		if ( !isRequired && !( [ "many-to-many", "one-to-many" ] ).findNoCase( relationship ) ) {
+		if ( !isRequired && !( [ "many-to-many", "one-to-many" ] ).findNoCase( relationship ) && !Len( Trim( propertyDefinition.formula ?: "" ) ) ) {
```

Comment:
Can we set an `isFormula` variable once at the top of the method and then use that throughout?